src/universal/auth/pages/ChangePasswordPage/ChangePasswordPage.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
+import * as Helmet from 'react-helmet';
+import {FormattedMessage, Paper, FormattedLink} from 'universal/common/components';
+import {reset} from 'redux-form';
+import {changeUserPassword} from 'universal/auth/actions';
+import ChangePasswordForm, {CHANGE_PASSWORD_FORM_NAME} from './ChangePasswordForm';
+const {connect} = require('react-redux');
+const {Component, PropTypes} = React;
+
+@connect(
+  ({auth}) => ({
+    isChangingPassword: auth.isChangingPassword,
+    isChangePasswordSuccess: auth.isChangePasswordSuccess,
+    changePasswordError: auth.changePasswordError,
+  }),
+  {
+    reset,
+    changeUserPassword,
+  })
+export default class ChangePasswordPage extends Component<any, any> {
+  public static propTypes = {
+    isChangingPassword: PropTypes.bool.isRequired,
+    changePasswordError: PropTypes.string.isRequired,
+    reset: PropTypes.func.isRequired,
+    changeUserPassword: PropTypes.func.isRequired,
+  };
+
+  public async handleSubmit(data) {
+    const {password} = data;
+    const userId = this.props.params.userId;
+    const resetPasswordId = this.props.params.resetPasswordId;
+
+    if (password) {
+      const response = await this.props.changeUserPassword(password, userId, resetPasswordId);
+      if (response && response.result) {
+        this.props.reset(CHANGE_PASSWORD_FORM_NAME);
+      }
+    }
+  }
+
+  public render() {
+    const css = require('./ChangePasswordPage.scss');
+    const component = this.getComponent(css);
+
+    return (
+      <Paper className={css.ChangePasswordPage}>
+        <Helmet title="EverReal - change password" />
+        <h3><FormattedMessage id="ChangePasswordPage.page.title" /></h3>
+
+        {component}
+      </Paper>
+    );
+  }
+
+  private getComponent(css) {
```

Comment:
This will be fixed as removed getComponent and moved the logic to the reducer. But this would still be an issue, maybe open a ticket?